Set the default block size on the BitcoinCash chain to 2MB

ptschip opened the pull request:
Also, set the block priority area to 5% of the default blocksize

ptschip committed:
Define the default mining block size in BITCOIN_CASH to be 2MB

ptschip committed:
Make the maxGenerateBlock to be the DEFAULT_BLOCK_MAX_SIZE

ptschip committed:
Fix log message

sickpig commented on src/unlimited.cpp:
```diff
@@ -63,7 +63,7 @@ UniValue validatechainhistory(const UniValue &params, bool fHelp);
 {
     // The mined block size must be less then or equal too the excessive block size.
-    LogPrintf("newMiningBlockSizei: %d - newExcessiveBlockSize: %d", newMiningBlockSize, newExcessiveBlockSize);
+    LogPrintf("newMiningBlockSize: %d - newExcessiveBlockSize: %d\n", newMiningBlockSize, newExcessiveBlockSize);
```
good catch :+1:

gandrewstone commented on src/policy/policy.h:
```diff
@@ -15,10 +16,20 @@
+static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = DEFAULT_BLOCK_MAX_SIZE / 20;
+#else
 static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 50000;
+#endif
```
since 1000000/20 = 50000 shouldn't we not #ifdef this?

ptschip committed:
Remove #ifdef BITCOIN_CASH for blockpriority space

Make blockprioritysize 5% of the default blocksize
regardless of implementation.

ptschip commented:
@gandrewstone good point...changes pushed up.
